upload.py

```diff
@@ -136,13 +136,14 @@ def check_missing_data(asset_type, props):
                 "   Proper categorization significantly improves your asset's discoverability.",
             )
 
-    thumb_path = bpy.path.abspath(props.thumbnail)
-    if props.thumbnail != "" and not os.path.exists(Path(thumb_path)):
-        write_to_report(
-            props,
-            "Thumbnail filepath does not exist on the disk.\n"
-            "   Please check the filepath and try again.",
-        )
+    if asset_type != "HDR":
```

Comment:
HDRs nemaji props.thumbnail. Jak tomu rozumim, loadujeme to primo z <bpy_struct, Image(".4193dc9c-9282-408a-b793-b693e6ad2e5c"), takze vlastne je to uz nactene a nema smysl kontrolovat, zda je to na disku, je to spravne?

Jestli se muze stat, ze bpy_struct Image je prazdny, tak mi dej @vilemduha vedet a stalo by za to pridat kontrolu. Ale jak tomu ted rozumim, tak se to stat nemuze.

Reply:
Je to tak